File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/interfaces/IBlackList.java
package org.nirvanasms.api.core.interfaces;

import org.nirvanasms.api.core.entity.ApiUser;
import org.nirvanasms.api.core.entity.PhoneNumber;
import org.nirvanasms.api.core.exceptions.NonValidPhoneNumberException;
import org.nirvanasms.api.core.utility.ProcessResult;

public interface IBlackList {
	ProcessResult<?> add(ApiUser user, PhoneNumber phoneNumber) throws NonValidPhoneNumberException;
}

File: NirvanaSMS.Api.CSharp/Core/Interface/IBlackList.cs
using NirvanaSMS.Api.Core.Entity;
using NirvanaSMS.Api.Core.Utility;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NirvanaSMS.Api.Business.Interface
{
    interface IBlackList
    {
        ProcessResult Add(ApiUser user, PhoneNumber phoneNumber);
    }
}
